From 9c36494960fbff6a19df3a9e8b399ea0ec939ba4 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Sun, 14 May 2017 22:39:34 -0400
Subject: [PATCH 015/103] stop hiding SLUB_DEBUG behind EXPERT

It can make sense to disable this to reduce attack surface / complexity.
---
 init/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init/Kconfig b/init/Kconfig
index ae774d17fe59..c3749ea82cf0 100644
--- a/init/Kconfig
+++ b/init/Kconfig
@@ -1764,7 +1764,7 @@ config VM_EVENT_COUNTERS
 
 config SLUB_DEBUG
 	default y
-	bool "Enable SLUB debugging support" if EXPERT
+	bool "Enable SLUB debugging support"
 	depends on SLUB && SYSFS
 	help
 	  SLUB has extensive debug support features. Disabling these can
-- 
2.28.0

